db/migrate/20200407212050_add_column_role_to_users.rb

```diff
@@ -1,5 +1,5 @@
 class AddColumnRoleToUsers < ActiveRecord::Migration[5.1]
   def change
-    add_column :users, :role, :integer, default: 0
+    add_column :users, :role, :integer, default: 1
```

Comment:
Why did you change the default role?

Reply:
the fact that a user object exists means they are at least role :1 (default). I think it makes sense to make the default user_role 1,